README.md

```diff
@@ -16,6 +16,13 @@ or if you want to build your image then jump straight into analyzing it:
 dive build -t <some-tag> .
 ```
 
+Additionally you can run this in your CI pipeline to ensure you're keeping wasted space to a minimum (this skips the TUI):
```

Comment:
Did you mean `this skips the UI` not `this skips the TUI`?

Reply:
TUI = terminal UI, but UI makes more sense universally